hw/dv/data/riviera/riviera.hjson

```diff
@@ -0,0 +1,84 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "{job_prefix} vlog"
```

Comment:
```suggestion
  build_cmd:  "vlib work && {job_prefix} vlog"
```
The `hw/dv/data/sim.mk` needs to remain as simulator agnostic and lean as possible.